packages/graphql/src/schema/make-augmented-schema.ts

```diff
@@ -489,7 +489,7 @@ function makeAugmentedSchema(
 
         const nodeInput = composer.createInputTC({
             name: `${node.name}CreateInput`,
-            // TODO - Duplicated with rel properties
+            // TODO - Duplicated with relationship properties on line 313
```

Comment:
I think in general we should refrain from saying things like "on line 313". For example, we did something similar in the eslint file "(approximately 350 instances)" and it's unrealistic for the numbers to be in check and could mislead others.